chaincfg/init_test.go

```diff
@@ -121,21 +121,21 @@ var (
 				Id:          "Abstain",
 				Description: "Abstain voting for Pedro",
 				Bits:        0x0,   // 0b0000
-				IsIgnore:    false, // XXX this is the invalid bit
+				IsAbstain:   false, // XXX this is the invalid bit
```

Comment:
It's a test to ensure a vote that has the vote bits set to 0, but is not properly marked as an abstaining vote causes an error.